res/nvimrc

```diff
@@ -77,7 +99,8 @@ anoremenu .900 &Window.Toggle\ Full\ Screen :call MacToggleFullScreen()<CR>
 " Assign Cmd-key shortcuts
 call MacMenu("File.New Window", "T-n")
 call MacMenu("File.Save", "T-s")
-call MacMenu("File.Open...", "T-o")
+call MacMenu("File.Save As", "T-S-s")
+call MacMenu("File.Open", "T-o")
```

Comment:
Don't remove the ... from File.Open — it signifies that the menu item opens a dialog. Save As should have one too.